src/main/scala/GameRunner.scala

```diff
@@ -0,0 +1,113 @@
+import GameHelpers._
+import GamePrompts._
+import scala.util.Random
+
+object GameRunner extends App {
+
+  def gameStart(): Unit = {
+    println()
+    println("Welcome to \u001b[31;1mScala Scrabble Practice")
+    println()
+    val name: String = promptName()
+    println()
+    println("Hello " + name + ". \u001b[0mThe Human plays against the Computer.")
+    println()
+    val level: String = promptLevel()
+
+    val letterBag = Bag.initialise()
+    val human = Player(name, Nil, Nil, false)
```

Comment:
In the same way that you have a `Bag.initialise`, why not create one for `Player` to save yourself some boilerplate? Perhaps

```scala
def apply(???): Player = ???
```